src/views/shared/AuthContainer.js

```diff
@@ -14,9 +14,10 @@ import { View } from 'okta';
 import Enums from 'util/Enums';
 const CAN_REMOVE_BEACON_CLS = 'can-remove-beacon';
 export default View.extend({
+  tagName: 'main',
```

Comment:
https://caniuse.com/?search=main
No IE 11 support.. 🤦

Reply:
Solution: https://stackoverflow.com/questions/35820429/main-element-not-working-in-internet-explorer-11#:~:text=1%20Answer&text=The%20HTML5%20main%20element%20is,element%20for%20width%20to%20work.